html/controls/plt/plt-input.js

```diff
@@ -29,7 +29,11 @@ function preparePltInputTable(element, teamId, mode, statsbookPeriod, alternateN
 			$('<td>').text('Pivot').appendTo(thead);
 			$('<td>').text('Blocker').appendTo(thead);
 			$('<td>').text('Box').appendTo(thead);
-			$('<td>').appendTo(thead);
+			if (mode == 'lt') {
+				$('<td>').attr('id', 'head').text('Team ' + teamId).appendTo(thead);
```

Comment:
This field is to small for a long team name.
I'd rather use it for the SP indicator (on PLT as well).

Reply:
You're probably right, but it'll do the job for now.